include/checksec.h

```diff
@@ -60,8 +60,9 @@ constexpr const char kAuthenticodeDescription[] =
     "Binaries with Authenticode signatures are verified at load time.";
 
 constexpr const char kRFGDescription[] =
-    "Binaries with RFG enabled have additional return-oriented-programming "
-    "protections.";
+    // https://www.techrepublic.com/article/windows-10-security-how-the-shadow-stack-will-help-to-keep-the-hackers-at-bay/
```

Comment:
This link is pretty light on details, consider replacing it with this one: https://xlab.tencent.com/en/2016/11/02/return-flow-guard/

Also, please put it above the variable declaration 🙂